<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmCloseDialog</class>
 <widget class="QDialog" name="ConfirmCloseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Changes</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="dontSaveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save changes before closing?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
